src/components/CustomCalendar/useCalendar.tsx

```diff
@@ -8,7 +8,13 @@ import { StyledWeekday } from './styles';
 
 import { SHORT_MONTH, SHORT_WEEKDAY } from '@/constants/dateFormats';
 
-const useCalendar = (): {
+const useCalendar = ({
+  currentDate,
+  setCurrentDate,
+}: {
+  currentDate: Date;
+  setCurrentDate: Dispatch<SetStateAction<Date>>;
+}): {
   currentDate: Date;
```

Comment:
I think it's ok when we have 2 or 3 props, I'd better create interface for returned type

Reply:
Fixed